_posts/salads/2019-11-28-winter-squash-panzanella.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: Winter Squash Panzanella
+category: salads
+tags: recipeish
+---
+
+I made this salad for a Thanksgiving potluck at [Mapbox][mbx], wanting to make
+something vegan with the bounty of butternut squash from our CSA. Panzanella is
+definitely a 'pattern salad': a base of toasted (or stale) bread moistened and
+combined with other stuff. You can (and should!) replace the primary vegetable,
+toppings, and dressing with whatever is currently in-season. This recipe itself
+is based on the panzanella in Marcella Hazan's [Essentials of Classic Italian
+Cooking][hazan] (which is tomato and anchovy-based), and suggestions from
+friends at the [Garden for the Environment][gfe] at an event the night before
+the potluck.
+
+The salad components can be prepped the day before, *except* for the vegetable
+toppings (radishes, etc.). Those should be prepped at the time of serving.
+
+Salad dressing
+- 1 cup olive oil
+- 2 tbsp sherry vinegar
+- 2 tbsp red wine vinegar
+- 4 tbsp salt-packed capers
+- 2-3 cloves of garlic, mortared with the capers
+- 1 tsp whole-grain mustard
+
+Squash
+- 4 whole butternut squash, cut in-half and sliced into 1/2-inch half-moons
+- Toss with salt, olive oil, white pepper
+- Roast on parchment at 400F until caramelized at the edges
+
+Bread
+- 1 loaf sourdough, cut into 1-inch cubes, tossed lightly in olive oil, and
+  broiled to golden brown. I used a whole [Tartine][tartine] country loaf, but
+  any substantial, high-gluten broad loaf works.
+
+Toppings
+- shallots, sliced and macerated in apple cider vinegar
+- radishes, sliced
+- chives, diced
+- arugula, washed
+- roasted Whole Hazelnuts, chopped
+
+Assemble the salad 30-minutes prior to serving:
+1. In a large mixing bowl, toss and roughly combine the bread and the
```

Comment:
isn't clear that the bread should be broiled before this step.

Reply:
Fixed!